gatsby-config.js

```diff
@@ -271,12 +271,8 @@ module.exports = {
       url: 'https://developer-stage.adobe.com'
     },
     {
-      prefix: '/console/docs/guides',
+      prefix: '/developer-console/docs/guides',
```

Comment:
why do we have this stage url here ?

Reply:
This proxy only gets hit in the `develop` mode, when you start your server using `npm run dev`, otherwise it does not even take affect. This could be developer.adobe.com as well.

Reply:
Proxy urls now point to production (see https://github.com/AdobeDocs/adobe-io-events/pull/14/commits/a4712000cb1b011f1c31496e337d8defc3130fda).